asc2qr.sh

```diff
@@ -45,38 +45,27 @@ if [ ! -f ${asc_key} ]; then
 	exit 1
 fi
 
-# Create a temp file to use as a pattern for splitting the input key file.
-# This helps protect against file collisions in the current directory.
-export TMPDIR=""
-tmp_file=$(mktemp keyparts.XXXXXX)
-if [ $? -ne 0 ]; then
-	echo "failed to create temporary file"
-	exit 1
-fi
-
-# Split the key file into usable chunks that the QR encoder can consume
-split -b ${file_split_size} ${asc_key} "${tmp_file}."
+## Split the key file into usable chunks that the QR encoder can consume
+chunks=()
+while true; do
+    s=$( head -c ${max_qr_bytes} )
```

Comment:
When I use the standard 2800 value for max_qr_bytes on my RSA 4096-bit key ( 'test_key.asc' ), I get 3 chunks generated. When I diff the final 'mykey.asc' key file against my 'test_key.asc' key file after generating QR codes (with asc2qr.sh) and decoding the QR codes (with 'qr2asc.sh' - with my decode line which I have added as a comment to that file), then I get no difference. I went ahead and added a script to assist in testing, but unless your bash does not work like my bash, then I am not seeing what you are. I am using bash 4.3.46 on Linux 4.4 (Ubuntu 16.04). I have not tested on any other OS. What OS did you run your test on?